webapp/src/main/java/com/box/l10n/mojito/entity/ScheduledJob.java

```diff
@@ -0,0 +1,153 @@
+package com.box.l10n.mojito.entity;
+
+import com.box.l10n.mojito.json.ObjectMapper;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobProperties;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import java.util.Date;
+import org.hibernate.envers.Audited;
+
+@Audited
+@Entity
+@Table(name = "scheduled_job")
+public class ScheduledJob {
+  @Id private String id;
+
+  @ManyToOne
+  @JoinColumn(name = "repository_id")
+  private Repository repository;
+
+  @ManyToOne
+  @JoinColumn(name = "job_type")
+  private ScheduledJobTypeEntity jobType;
+
+  @Column(name = "cron")
+  private String cron;
+
+  @Transient private ScheduledJobProperties properties;
+
+  @Column(name = "properties")
+  private String propertiesString;
+
+  @ManyToOne
+  @JoinColumn(name = "job_status")
+  private ScheduledJobStatusEntity jobStatus;
+
+  @Column(name = "start_date")
+  private Date startDate;
+
+  @Column(name = "end_date")
+  private Date endDate;
+
+  @Column(name = "enabled", nullable = false, columnDefinition = "TINYINT DEFAULT 1")
```

Comment:
I don't see this column definition used in the repo before. Is it necessary?

Reply:
Also is the nullable false necessary since you default the field value to false?

Reply:
Ideally, we would use the "BIT" column type here - as that is the closest semantical equivalent for storing a bool in MySQL and is the smallest amount of space used.

https://dev.mysql.com/doc/refman/8.0/en/bit-type.html

Reply:
This is done @byronantak @garionpin

Reply:
I'm happy with the change. I'll leave resolving to George to be sure he's happy with it